packages/tokens/bin/lib/formats/formatSvg.ts

```diff
@@ -0,0 +1,51 @@
+import {camelCase} from 'lodash';
+import {optimize, OptimizedSvg} from 'svgo';
+
+import {isTokenEnum, isTokenGroup, Token, TokenEnum, TokenGroup, TokenList} from './token';
+
+export type SvgFormatOutput = {fileName: string; svgMarkup: string};
```

Comment:
I find it used only in this file you can delete the export

```suggestion
type SvgFormatOutput = {fileName: string; svgMarkup: string};
```